schemas/public/ekscluster-kfd-v1alpha2.json

```
@@ -1266,27 +1316,28 @@
     "Spec.Distribution.Modules.Ingress.CertManager.ClusterIssuer": {
       "type": "object",
       "additionalProperties": false,
+      "description": "Configuration for the cert-manager's ACME clusterIssuer used to request certificates from Let's Encrypt.",
       "properties": {
         "name": {
           "type": "string",
-          "description": "The name of the cluster issuer"
+          "description": "Name of the clusterIssuer."
```

```suggestion
          "description": "The name of the clusterIssuer."
```
Just to keep the same wording as the other fields